ad-insertion/video-analytics-service/app/modules/ModelManager.py

```diff
@@ -80,3 +147,19 @@ def get_loaded_models():
                         results.append(result)
         return results
 
+if __name__ == '__main__':
```

Comment:
This looks like it was here for debugging?

Reply:
done